find.cc

```diff
@@ -788,7 +788,7 @@ class FindEmulatorImpl : public FindEmulator {
     }
 
     const size_t orig_out_size = out->size();
-    for (StringPiece finddir : fc.finddirs) {
+    for (string finddir : fc.finddirs) {
```

Comment:
Done (we still copy the string in this loop, and did before this change, but there are a couple of conditions first)